package.json

```diff
@@ -57,6 +57,6 @@
     "vitest": "^1.4.0"
   },
   "volta": {
-    "node": "18.17.1"
+    "node": "20.12.0"
```

Comment:
Better to put `"20"` as we want to use the latest of "20", not any version in particular.